floor/test/integration/inheritance/entity_inheritance_test.dart

```diff
@@ -0,0 +1,104 @@
+import 'dart:async';
+
+import 'package:floor/floor.dart';
+import 'package:floor_annotation/floor_annotation.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:sqflite/sqflite.dart' as sqflite;
+
+part 'entity_inheritance_test.g.dart';
+
+void main() {
+  group('dao inheritance tests', () {
```

Comment:
⚠️ The name doesn't match the test group

Reply:
whoops